### PR TITLE
docs(snap): update snap/README and add snap section in README

### PR DESCRIPTION
**Pull request by MonicaisHer**
**If your build fails** due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/device-modbus-go/blob/main/.github/Contributing.md

## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] I am not introducing a breaking change (if you are, flag in conventional commit message with `BREAKING CHANGE:` describing the break)
- [x] I am not introducing a new dependency (add notes below if you are)
- [ ] I have added unit tests for the new feature or bug fix (if not, why?)
- [ ] I have fully tested (add details below) this the new feature or bug fix (if not, why?)
- [ ] I have opened a PR for the related docs change (if not, why?)
  <link to docs PR>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,6 +20,14 @@ $ ./simulator
...
 Start up a Modbus TCP simulator.
 ```
+### Running the device-modbus Service as a Snap
+EdgeX Device modbus is also available as a snap package. Install the snap with the following command:
+```
+sudo snap install edgex-device-modbus
+```
+for more details on the Device modbus Snap, including installation, configuration, please refer to [EdgeX Modbus Device Service Snap](https://github.com/edgexfoundry/device-modbus-go/tree/main/snap)
+
+for more details on Snap, including EdgeX Snap, viewing logs, security services, please check [Getting Started with Snap](https://docs.edgexfoundry.org/2.0/getting-started/Ch-GettingStartedSnapUsers/)
```
Comment (farshidtz) [inline README.md:30]: 2x ~for~ For 

more details on ~Snap, including EdgeX Snap~ deploying EdgeX with Snaps, viewing logs

Comment (MonicaisHer) [reply README.md]: Updated. Thanks!

**Review by farshidtz — APPROVED** [review]
Thanks. Looks good.

Please squash+merge.

**Comment by MonicaisHer** [comment]
Could you please help me squash and merge it? Thank you. @siggiskulason

**Review by siggiskulason — APPROVED** [review]
LGTM

**Pull request merged**